packages/devtools_app/lib/src/framework/release_notes/README.md

```diff
@@ -1,9 +1,8 @@
-## Writing DevTools release notes
+## Generating Release notes
```

Comment:
I think this section should be reorganized. Something like this:

<information about release notes>

To update release notes:
 1. <Action. Helping information.>
 2. <Action. Helping information.>
 3. <Action. Helping information.>
 ...

The steps that requires to do something before doing previous step looks counter-user-friendly :)

Reply:
I definitely agree that this section is a bit convoluted, but for this PR I would like to just focus on changing the part that this PR touches.